docs/architecture.md

```diff
@@ -21,25 +21,25 @@ connections to downstream clusters. This means managed clusters can run in priva
 NATs. The only requirement is the cluster agent needs to be able to communicate with the
 Kubernetes API of the cluster running the Fleet manager. The one exception to this is if you use
 the [manager initiated](./manager-initiated.md) cluster registration flow.  This is not required, but
-an optional patter.
+an optional pattern.
 
 The cluster agents are not assumed to have an "always on" connection.  They will resume operation as
 soon as they can connect. Future enhancements will probably add the ability to schedule times of when
 the agent checks in, as it stands right now they will always attempt to connect.
 
 ## Security
 
-The Fleet manager dynamically creates service account, manages their RBAC and then gives the
+The Fleet manager dynamically creates service accounts, manages their RBAC and then gives the
 tokens to the downstream clusters. Clusters are registered by optionally expiring cluster registration tokens.
 The cluster registration token is used only during the registration process to generate a credential specific
 to that cluster. After the cluster credential is established the cluster "forgets" the cluster registration
  token.
 
 The service accounts given to the clusters only have privileges to list `BundleDeployment` in the namespace created
-specifically for that cluster.  It can also update the `status` subresource of `BundleDeployment` and the `status`
+specifically for that cluster. They can also update the `status` subresource of `BundleDeployment` and the `status`
```

Comment:
This is referring to multiple service accounts, and `It` is fine.

Reply:
Dropping it then :).